var/spack/repos/builtin/packages/mofem-cephas/package.py

```diff
@@ -0,0 +1,132 @@
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+
+from spack import *
+
+
+class MofemCephas(CMakePackage):
+    """mofem-cephas core library"""
+
+    @property
+    def root_cmakelists_dir(self):
+        """The relative path to the directory containing CMakeLists.txt
+
+        This path is relative to the root of the extracted tarball,
+        not to the ``build_directory``. Defaults to the current directory.
+
+        :return: directory containing CMakeLists.txt
+        """
+        return 'mofem'
+
+    homepage = "http://mofem.eng.gla.ac.uk"
+    url = "https://likask@bitbucket.org/likask/mofem-cephas.git"
+
+    version('0.8.2', git='https://likask@bitbucket.org/likask/mofem-cephas.git',
+        tag='v0.8.2', submodules=True)
+    version('0.8.1', git='https://likask@bitbucket.org/likask/mofem-cephas.git',
+        tag='v0.8.1', submodules=True)
+    version('0.8.0', git='https://likask@bitbucket.org/likask/mofem-cephas.git',
+        tag='v0.8.0', submodules=True)
+    version('0.7.29', git='https://likask@bitbucket.org/likask/mofem-cephas.git',
+        tag='v0.7.29')
+    version('0.7.28', git='https://likask@bitbucket.org/likask/mofem-cephas.git',
+        tag='v0.7.28')
+    version('0.7.27', git='https://likask@bitbucket.org/likask/mofem-cephas.git',
+        tag='v0.7.27')
+    version('develop',
+        git='https://likask@bitbucket.org/likask/mofem-cephas.git',
+        branch='develop')
+
+    # Variants of packages installed as extensions with MoFEM, to indicate this
+    # specific type of variant prefox with_ is added.
+    variant('with_adol-c', default=False,
+            description='Install ADOL-C with MoFEM')
+    variant('with_tetgen', default=False,
+            description='Install TetGen with MoFEM')
+    variant('with_med', default=False,
+            description='Install MED with MoFEM')
+
+    # This option cab ne only used for development of core lib
+    variant('copy_user_modules', default=True,
+        description='Copy user modules directory instead linking to source')
+    variant('adol-c', default=True, description='Compile with Adol-C')
+    variant('tetgen', default=True, description='Compile with Tetgen')
+    variant('med', default=True, description='Compile with Med')
+    variant('slepc', default=True, description='Compile with Slepc')
+    variant('doxygen', default=False, description='Install doxygen')
+
+    conflicts('+adol-c', when='+with_adol-c')
+    conflicts('+tetgen', when='+with_tetgen')
+    conflicts('+med', when='+with_med')
+
+    depends_on("mpi")
+    depends_on("parmetis")
+    # Fixed version of hdf5,
+    # to remove some problems with dependent packages, f.e. MED format
+    depends_on("hdf5@:1.8.19+hl") 
+    depends_on("petsc@:3.9.2+mumps")
```

Comment:
since you depend on `mpi` unconditionally, you might add `+mpi` in petsc as well.